server/php-master.cpp

```diff
@@ -1317,10 +1319,46 @@ void check_and_instance_cache_try_swap_memory() {
   }
 }
 
+std::vector<statshouse::Metric> make_statshouse_metrics() {
```

Comment:
Now it's copypasted from here
https://github.com/VKCOM/kphp/blob/6dd46f3e709300c09590b1d96735ee86c2bc9dce/server/php-master.cpp#L1025

For test metrics it's ok, but at the next step is not appropriate, because in case of adding new metric you will need to do it in two places. So I think it should be done via patching `add_stat` function and adding new `stats_type`:
https://github.com/VKCOM/kphp/blob/6dd46f3e709300c09590b1d96735ee86c2bc9dce/common/stats/provider.cpp#L36